public/documentation/ios.md

```diff
@@ -0,0 +1,57 @@
+# Crosswalk Project for iOS
+
+## Introduction
+
+Crosswalk Project for iOS is a sub-project of Crosswalk, it aims to provide a web runtime to develop sophisticated iOS native or hybrid applications.
+
+* Extended WKWebView
+
+  Crosswalk Project for iOS is built on top of `WKWebView`, the mordern WebKit framework debuted in iOS 8.0. We extend the WKWebView to build Crosswalk extension framework within, and we are intended to bring the least intrusion to the interface to make developers use it as natural as WKWebView itself. For the detailed information you may refer to [Embedding Mode & WKWebView](/documentation/ios/embedding_mode_&_native_apis.html).
```

Comment:
Nice catch. The initial intention is that we won't introduce any new class, but just extend the WKWebView with a few interfaces methods & properties, to make user feel as if they are using the original version of WKWebView. So we call it "least intrusion" here.
But as we have to subclass from WKWebView and introduce the XWalkView as our interface class(due to the internal lifecycle issue we came across when using WKWebView extension way of implementation), this concept is no longer valid. I'll replace it with some other proper words here.